Prox/Prox/AppDelegate.swift

```diff
@@ -179,7 +184,12 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
                                 willPresent notification: UNNotification,
                                 withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
         // show a badge.
-        completionHandler(UNNotificationPresentationOptions.alert)
+        if let eventKey = notification.request.content.userInfo[notificationEventIDKey] as? String,
+            let placeKey = notification.request.content.userInfo[notificationEventPlaceIDKey] as? String {
+            Analytics.logEvent(event: AnalyticsEvent.EVENT_NOTIFICATION, params: [:])
```

Comment:
I wonder if we want to add a parameter to the flurry event to differentiate between remote/local notifications